cookbook/add-tabs.rst

```diff
@@ -0,0 +1,300 @@
+Adding tabs to Sulu admin interface
+===================================
+
+.. note:: It is recommended to read `Extend admin Ui`_ beforehand to get a better understandig of how Sulu admin classes work.
+
+
+This tutorial will walk you through the process of adding extra tabs to the user interface of Sulu.
+
+This could be useful in a few situations, a few examples below:
```

Comment:
The examples are almost the same... Could we maybe write something like this without a listing?

"This could be useful in many different situations, like e.g. adding an extra tab for social media information to pages or contacts."